templates/0.12.0/main/src/main.rs.gdpu

```diff
@@ -35,7 +34,7 @@ fn main() -> amethyst::Result<()> {
         )?
         .with_bundle(TransformBundle::new())?;
 
-    let mut game = Application::new("/", MyState, game_data)?;
+    let mut game = Application::new(app_root.join("assets"), MyState, game_data)?;
```

Comment:
Sorry I'm coming in late, but could this be a part of a separate PR for updating 0.12.0, at least just for tracking purposes?